en/advanced_config/bootloader_update.md

```diff
@@ -0,0 +1,27 @@
+# Bootloader Update
+
+Early [Pixhawk-series](../flight_controller/pixhawk_series.md#fmu-versions) flight controllers based on FMUv2 had a hardware issue ([Silicon Errata](../flight_controller/silicon_errata.md#fmuv2--pixhawk-silicon-errata)) that restricted them to using 1MB of flash memory. 
+The problem was fixed on newer hardware, so it can now (in theory) install FMUv3 Firmware and access all 2Mb available memory.
+
+Unfortunately some boards come from the factory with an old bootloader that is unable to detect whether or not the hardware issue is present. 
+As a result, the memory-restricted FMUv2 Firmware must still be used.
+
+This topic explains how you can update the bootloader to the latest version so that you can use FMUv3 Firmware on compatible boards.
+
+
+### Main Steps
+
+You can initiate bootloader update on next restart by [setting the parameter](../advanced_config/parameters.md#parameter-configuration): [SYS_BL_UPDATE](../advanced_config/parameter_reference.md#SYS_BL_UPDATE).
+
+To update the bootloader:
+
+1. Insert an SD card (enables boot logging to debug any problems.)
+1. Flash master of `fmu-v2 target` (e.g. through QGC)
+   ![FMUv2 update](https://user-images.githubusercontent.com/13203106/45455962-cff71f80-b69d-11e8-8b90-cfce2be689b1.JPG)
+1. Enable [SYS_BL_UPDATE](../advanced_config/parameter_reference.md#SYS_BL_UPDATE)
+1. Reboot & Wait.
```

Comment:
@bkueng After updating, rebooting and waiting, does the system automatically go to firmware update or does it go to airframe? 

Ie Normally after an update you'll go to airframe, so below I assume you have to disconnect and reconnect and then do firmware update as per usual.?

Also, can you confirm that in this update you'd just select latest stable build, and the FMUv3 firmware will be picked if it is valid (and not if the errata affects the hardware)

Reply:
QGC will do whatever it would do normally (I don't update through QGC :)

> Also, can you confirm that in this update you'd just select latest stable build, and the FMUv3 firmware will be picked if it is valid (and not if the errata affects the hardware)

Yes it should, but there might be a bug in the detection (https://github.com/PX4/Firmware/pull/10426#issuecomment-420820319).

Reply:
I'll assume that is fixed before you accept the PR :-)